### ISSUE TITLE
Freeze when geocoding using OSM Nominatim 

### ISSUE THREAD

**Issue by lmu-soz** [michaelminn/mmqgis#24]
I am using the MMQGIS on QGIS 3.14.0 with OSM Nominatim. But whenever I run the process, the program freezes. And it cannot find any addresses of the CSV UTF-8 file. There are no null values in any fields and I cannot find out what causes this problem.

![grafik](https://user-images.githubusercontent.com/69844618/90502558-53550180-e14e-11ea-8766-8d362db8f210.png)

Dos anyone have the same experience or any ideas on how to fix it?

**Comment by lmu-soz** [comment]
**Problem solved!**
The sequence in the address field was wrong. In Germany we format the addresses "street no" and not the other way around.
I changed it and now it does not freeze anymore. :)

**Comment by lefcgis** [comment]
Yep. You must 'normalized' your data previous. Then, you can apply it.
